changelog_unreleased/javascript/11750.md

```diff
@@ -1,4 +1,4 @@
-#### Fix errors on deleting variables (#11750 by @fisker)
+#### Ignore errors for non-strict mode features (#11750 by @fisker, #11778 by @sosukesuzuki)
```

Comment:
these features are not important so I think we don't need to add more code examples.

Reply:
```suggestion
#### Ignore errors for sloppy mode syntax (#11750 by @fisker, #11778 by @sosukesuzuki)
```